roles/etcd/tasks/main.yml

```diff
@@ -120,7 +120,9 @@
     enabled: yes
   register: start_result
 
-- include: etcdctl.yml
+- include_role:
+    name: etcd_common
+    tasks_from: etcdctl.yml
```

Comment:
Since the `etcdctl.yml` installs etcd and that should be done right after the etcd cluster is actually installed and configured. Setting `etcd_common/tasks/main.yml` -> `etcd_common/tasks/main.yml ` will first install etcd [1] and then run the etcd role.

[1] https://github.com/openshift/openshift-ansible/blob/ba5c6594ade6679d055aaecc7c38836444f0f791/roles/etcd/tasks/etcdctl.yml#L2